Jenkinsfile

```diff
@@ -0,0 +1,117 @@
+pipeline {
+    agent {
+        label 'linux && python'
+    }
+    stages {
+        stage('Build') {
+            steps {
+                echo 'Building..'
+                sh '''
+                    virtualenv ./sdk-deploy
```

Comment:
specify specific python version. Consider using tox to set up the testing matrix.